app/src/Submit/OnlineJudgeFactory/OnlineJudge.ts

```diff
@@ -27,7 +27,8 @@ import open from "open";
 
 export enum OnlineJudgeName {
   codeforces = "codeforces",
-  atcoder = "atcoder"
+  atcoder = "atcoder",
+  omegaup = "omegaup"
 }
```

Comment:
every online judge name should be declared in this enum, and use this one, instead of raw strings